spec/board_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "./lib/board.rb"
+
+describe Scrabble::Board do
+  before :each do
+    @board = Scrabble::Board.new
+  end
+
+  describe 'initialize' do
+    it "nake a matrix array of 15" do
+      expect(@board.board).to be_an(Array)
+    end
+
+  describe 'initialize board' do
```

Comment:
Should also test that the board is a 15x15 matrix by checking that each element of `@board.board` is an `Array` and has a length of 15.